cmd/state-svc/internal/rtwatcher/watcher.go

```diff
@@ -98,6 +105,39 @@ func (w *Watcher) check() {
 func (w *Watcher) RecordUsage(e entry) {
 	logging.Debug("Recording usage of %s (%d)", e.Exec, e.PID)
 	w.an.Event(anaConst.CatRuntimeUsage, anaConst.ActRuntimeHeartbeat, e.Dims)
+	w.UpdateLastUsed(e)
+}
+
+func (w *Watcher) UpdateLastUsed(e entry) {
+	logging.Debug("Updating last usage of project that contains %s", e.Exec)
+	localProjects := projectfile.GetProjectMapping(w.cfg)
+	for namespace, checkouts := range localProjects {
+		for _, checkout := range checkouts {
+			proj, err := project.FromPath(checkout)
+			if err != nil {
+				multilog.Error("Unable to get project %s from checkout: %v", checkout, err)
+				continue
+			}
+			projectTarget := target.NewProjectTarget(proj, nil, "")
+			execDir := setup.ExecDir(projectTarget.Dir())
```

Comment:
```suggestion
			execDir := setup.ExecDir(proj.Dir())
```

Reply:
Nope, that will return the activestate.yaml directory with "exec" tacked on (I tried). I need the runtime exec directory.